src/App.js

```diff
@@ -1,19 +1,71 @@
 import "./App.scss";
+import React, { useState } from "react";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
+
+import WeatherIcon from "./components/Picture/WeatherIcon";
+import CurrentWeather from "./components/CurrentWeather/CurrentWeather";
+import FutureWeather from "./components/FutureForecast";
+
+
 import Icon from './components/Icon/Icon';
 
-//configs
 const siteTitle = process.env.REACT_APP_SITE_TITLE ?? "CYF Weather";
 
 function App() {
+  const [weatherData, setWeatherData] = useState([]);
+  
+  function getNewWeather(city) {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=metric&appid=3b86046cce0de3be7cfa8369f4540b37`
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        setWeatherData(data);
+      });
+  }
+  
   return (
     <div className="App">
-      <Header title={siteTitle} />
+      <Header
+        title={siteTitle}
+        getNewWeather={getNewWeather}
+        weatherData={weatherData}
+      />
+
       <main className="c-site-main" tabIndex="0">
+
+        <section>
+          {/* <WeatherIcon weatherId={weatherData?.weather?.[0]?.id} /> */}
+          {weatherData?.list?.map((icon) => (
+            <WeatherIcon weatherId={icon.weather[0].id} />
+          ))}
+
+          {weatherData?.list?.map((current) => (
+            <CurrentWeather
+              description={current.weather.description}
+              temp={current.main.temp.toFixed()}
+              humidity={current.main.humidity}
+              pressure={current.main.pressure}
+            />
+          ))}
+        </section>
+
+        <section>
+          {weatherData?.list?.map((future) => (
+            <FutureWeather
+              time={future.dt_txt}
+              iconId={future.weather[0].id}
+              temp={future.main.temp.toFixed()}
+            />
+          ))}
```

Comment:
Here we do want more than one item, however:

1. We don't want _all_ the data available. There are 40 items in the API data, but we only want to display 7 items (we know this because we are following the design. Always use the design to decide what your code needs to do).
2. It would be best to pass the future weather data to the `FutureWeather` component, and the component itself should `.map()` over it and display each item. Rather than mapping here inside App.js.

Reply:
1. I could mange to display 7 items. But positioning is incorrect and I am not sure whether I should use CSS grid instead of flexbox. 
2. I tried to pass the future weather data to the FutureWeather component as you suggested but again I couldn't fetch data from API in child component.

Reply:
1. You can use grid but you should be able to achieve the layout with just flexbox. It's up to you. The flex container should be full width but have some padding on the sides, so it won't actually look completely full width which is what we want.
The flex items should be placed in a row, centered and with space-between.

Then within each item, you should use flexbox again, this time to place the time, icon and temperature in a column. Again centered. 

2. You do not need to call the API again! The whole point is to call it once in the parent, and then pass the array of 7 weather items to the FutureWeather component as a prop.
So you should add a prop called for example 'weatherItems' to the FutureWeather component, with your array of 7 items as the value.
Then inside FutureWeather you will receive props.weatherItems and .map() over it (since it is an array). And for each item in the map you can display your weather item with time, icon and temp. 

Hope this makes sense! Give it a go and I'll have a look when you're ready.